test/support/factory.ex

```diff
@@ -67,6 +67,19 @@ defmodule Livebook.Factory do
     }
   end
 
+  def build(:fs_s3) do
+    bucket_url = "https://mybucket.s3.amazonaws.com"
+    hash = :crypto.hash(:sha256, bucket_url)
+
+    %Livebook.FileSystem.S3{
+      id: "s3-#{Base.url_encode64(hash, padding: false)}",
```

Comment:
Duplicating the id logic feels brittle, we can just call `Livebook.FileSystem.S3.new` right?

Reply:
Yeah, I'll rollback the `new` function and use it

Reply:
🐈‍⬛